package.json

```diff
@@ -20,61 +20,84 @@
   "files": [
     "build"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ydb-platform/ydb-ui-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ydb-platform/ydb-ui-components/issues"
+  },
   "scripts": {
     "clean": "gulp clean",
     "build": "gulp",
     "test": "echo \"Error: no test specified\" && exit 0",
     "lint:js": "eslint --ext .js,.jsx,.ts,.tsx .",
     "lint:styles": "stylelint 'src/**/*.scss'",
+    "lint:other": "npm run prettier -- --check",
```

Comment:
Is `-- --` intentional? What does it mean?

Reply:
this is how npm passes parameters to the script being executed